src/Traits/NormalizesPath.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Spatie\Export\Traits;
+
+use Illuminate\Support\Str;
+
+trait NormalizesPath
```

Comment:
Could this be renamed to `normalizedPaths`?

Reply:
Wasn't the rule that traits are named by the behavior thy add?
But for sure I can rename it.

Reply:
Sorry, apparently I completely derped while typing this comment. What I meant was `NormalizesPaths`. I'll change it myself after merging in a sec :)